HelpSource/Classes/Recorder.schelp

```diff
@@ -0,0 +1,144 @@
+class:: Recorder
+summary:: Write Audio to Harddisk
+categories:: Server>Abstractions
+related:: Classes/Server, Classes/DiskOut, Guides/Non-Realtime-Synthesis
+
+description:: A Recorder allows you to write audio to harddisk, reading from a given bus and a certain number of channels, relative to a given node. A link::Classes/Server:: has one instance, which is accessible also through the link::Classes/ScIDE::. You can use the server directly to record its output
+
+
+code::
+(
+{ SinOsc.ar(
+	SinOsc.ar(
+		XLine.kr(1, 100, 5)).exprange(*XLine.kr([20, 800], [7000, 200], 10)
+    )
+   ) * 0.1
+
+}.play;
+s.record(duration: 10);
+)
+::
+
+
+This functionality is also available through the recording button on the server windows.
+Pressing it once calls record, and pressing it again calls stopRecording (see below). When doing so the file created will be in your recordings folder and be named for the current date and time.
+The default location of the recordings folder varies from platform to platform. Setting this variable allows you to change the default.
+
+code::
+// find where the recordings are written to
+thisProcess.platform.recordingsDir
+::
+
+NOTE::
+By default, record creates the recording synth after the Server's default group and uses In.ar. Thus if you add nodes after the recording synth their output will not be captured.
+To avoid this, either use Node objects (which use the default node as their target) or (when using messaging style) use a target nodeID of 1.
+code::
+s.sendMsg("/s_new", "default", s.nextNodeID, 1,1);
+::
+::
+
+For more detail on this subject see link::Guides/Order-of-execution::, link::Reference/default_group::, and link::Guides/NodeMessaging::.
+
+See link::Classes/SoundFile:: for information on the various sample and header formats.
+Not all sample and header formats are compatible. Note that the sampling rate of the output file will be the same as that of the server app. This can be set using the Server's link::Classes/ServerOptions::.
+
+
+
+ClassMethods::
+
+method::new
+Create a new instance for a given server.
+
+argument::server
+
+
+InstanceMethods::
```

Comment:
Two small doc warnings:

    WARNING: SCDoc: In /home/nathan/git/supercollider/HelpSource/Classes/Recorder.schelp
      Method -prepareForRecord has 2 args, but doc has 1 argument:: tags.
    WARNING: SCDoc: In /home/nathan/git/supercollider/HelpSource/Classes/Recorder.schelp
      Method -record has 5 args, but doc has 1 argument:: tags.